go-controller/pkg/config/config.go

```diff
@@ -906,6 +910,12 @@ var CommonFlags = []cli.Flag{
 		Value:       Default.Zone,
 		Destination: &cliConfig.Default.Zone,
 	},
+	&cli.UintFlag{
+		Name:        "max-user-defined-networks",
+		Usage:       "The maximun number of user defined network primary or secondary allowed",
```

Comment:
nit; change usage to: The maximum number of user defined networks that are allowed (inclusive of both primary and secondary)